base/server/python/pki/server/deployment/scriptlets/subsystem_layout.py

```diff
@@ -135,15 +135,17 @@ def destroy(self, deployer):
             deployer.directory.delete(
                 deployer.mdict['pki_subsystem_profiles_path'])
         deployer.directory.delete(deployer.mdict['pki_subsystem_path'])
-        # remove instance-based subsystem logs
-        if deployer.mdict['pki_subsystem'] in \
-                config.PKI_SIGNED_AUDIT_SUBSYSTEMS:
+        # remove instance-based subsystem logs only if --remove-logs flag is specified
+        if deployer.mdict['pki_remove_logs']:
+            if deployer.mdict['pki_subsystem'] in \
+                    config.PKI_SIGNED_AUDIT_SUBSYSTEMS:
```

Comment:
The check against PKI_SIGNED_AUDIT_SUBSYSTEMS is no longer useful since it will always be true. IIRC in the past it didn't include TPS. I think we can remove this in a separate patch.

Reply:
Sounds good! Thank you, @edewata  :+1: